app/src/main/java/com/xmartlabs/template/di/RestServiceModuleApi.kt

```diff
@@ -1,6 +1,5 @@
-package com.xmartlabs.template.module
+package com.xmartlabs.template.di
```

Comment:
haha you liked it

Reply:
yep,  hahah from my point of view, `module` is not fine, it's too generic

Reply:
agree

Reply:
it's like "variable"